pkg/sql/opt/xform/groupby_funcs.go

```diff
@@ -228,6 +228,10 @@ func (c *CustomFuncs) GenerateStreamingGroupByLimitOrderingHint(
 	groupingCols opt.ColSet,
 	newOrdering props.OrderingChoice,
 ) {
+	if !c.e.evalCtx.SessionData().OptimizerUseLimitOrderingForStreamingGroupBy {
```

Comment:
nit: add an xform test with this setting disabled and an `expect-not` to verify this works